README.md

```diff
@@ -1,13 +1,11 @@
 
-<img width="100%" alt="pipeline" src="https://github.com/user-attachments/assets/11ebfb8d-95d7-43f1-b95d-2f14bf4dac86">
+<img width="100%" alt="pipeline" src="https://github.com/user-attachments/assets/eee02d08-9c76-4a6d-a103-812d7cac81a1">
 <p>
 
-- https://observablehq.com/@jstonge/allotaxonometer-for-all-exp
-- https://allotaxp.vercel.app/
-- :octocat: https://github.com/jstonge/allotaxp
-- :octocat: https://github.com/carterwward/py-allotax
+- Visit our [single-page web application](https://allotax.vercel.app/) to try it out online ([:octocat: github](https://github.com/jstonge/allotax))!
+- Use [py-allotax](https://github.com/carterwward/py-allotax) to use allotaxonometry at scale ([:octocat: github](https://github.com/carterwward/py-allotax))
```

Comment:
Move the tools under the description of "Allotaxonometer.js is a lightweight ..."